portfolio/src/main/webapp/index.html

```diff
@@ -97,7 +92,7 @@ <h2>Featured Recipes</h2>
               <a href="./recipe-3.html">
                 <img src="../images/rice.jpg" alt="Stir-fried Rice">
               </a>
-              <div class="desc">Lifeng's Famous Stir-Fried Rice</div>
```

Comment:
Changed the naming so that all recipes had a similar title length (better for homepage display)